src/components/Login/Login.jsx

```diff
@@ -1,9 +1,54 @@
 import React from 'react';
+import { connect } from 'react-redux';
```

Comment:
This component and the `Register` component need to be refactored to reflect the new state of the `Auth` component

Reply:
We need `connect` to get the `success` state from the redux store